### PR TITLE
Sc-8433: member detail feature

### PR DESCRIPTION
**Pull request by masskoder**
### Scope of changes

This adds the Member detail feature. The user could click on the detail icon and see the member information on the modal.

 
### Type of change

- [ ] bug fix
- [x] new feature
- [ ] documentation
- [ ] other (describe)

### Acceptance criteria

https://www.awesomescreenshot.com/video/18926370?key=e24dcd30724fff9b0b59c772b73cf4d9

### Author checklist

- [ ] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [x]  I have added new test fixtures as needed to support added tests
- [x]   Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [ ]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.

### CODE REVIEW

File: web/gds-user-ui/src/components/Loader/index.tsx [hunk 1]
```diff
@@ -13,7 +14,7 @@ const Loader: React.FC<LoaderProps> = ({ text, h, ...rest }) => {
       justifyContent={'center'}>
       <VStack spacing={4}>
         <Spinner color="blue.500" size="xl" {...rest} />
-        <Text>{text}</Text>
+        {withoutText ? null : <Text>{text}</Text>}
```
Comment (daniellemaxwell) [inline web/gds-user-ui/src/components/Loader/index.tsx:17]: Thanks for making this change!

**Review by daniellemaxwell — APPROVED** [review]
I made some minor suggestions/questions before the last commit was pushed. If you could take a look, I'd appreciate it. The modal looks good and I see that you were able to add in the contacts info. Good work!

File: web/gds-user-ui/src/modules/dashboard/member/components/MemberNetworkSelect.tsx [hunk 2]
```diff
@@ -1,8 +1,15 @@
 import { FormControl, FormLabel, Select } from '@chakra-ui/react';
 import { Trans } from '@lingui/macro';
+// import { Dispatch } from '@reduxjs/toolkit';
```
[the remark this answers is not in the document]
Comment (masskoder) [reply web/gds-user-ui/src/modules/dashboard/member/components/MemberNetworkSelect.tsx:3]: I added this to help you with the select network PR

File: web/gds-user-ui/src/modules/dashboard/member/components/MemberModal/MemberModalContent.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+const MemberDetail = ({ member }: MemberDetailProps) => {
+  return (
+    <>
+      <Stack w="full">
+        <SimpleGrid minChildWidth="120px" spacing="40px">
+          <Stack border={'1px solid #eee'} p={4} my={5} px={7} bg={'white'} id={'org'}>
+            <Box pb={5}>
+              <SimpleGrid minChildWidth="280px" spacing="40px">
+                <List>
+                  <ListItem fontWeight={'bold'}>
```
[the remark this answers is not in the document]
Comment (masskoder) [reply web/gds-user-ui/src/modules/dashboard/member/components/MemberModal/MemberModalContent.tsx]: Sure, I change it to be the same as all other views that have the same view to keep consistency on everything. I will notify Edwin about these changes when he will be back.  I forgot to add this note to the PR description.

You can see on the overview page how it has been done. 
image

File: web/gds-user-ui/src/modules/dashboard/member/components/MemberModal/MemberModalContent.tsx [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+                </List>
+                <List>
+                  <ListItem fontWeight={'bold'}>
+                    <Trans>Vasp Category</Trans>
+                  </ListItem>
+                  <ListItem>
+                    {member?.summary?.vasp_categories && member?.summary?.vasp_categories.length > 0
+                      ? member?.summary?.vasp_categories?.map((categ: any, index: any) => {
+                          return (
+                            <Tag key={index} color={'white'} bg={'blue'} mr={2} mb={1}>
```
[the remark this answers is not in the document]
Comment (masskoder) [reply web/gds-user-ui/src/modules/dashboard/member/components/MemberModal/MemberModalContent.tsx]: yes, the Tag element will do the same as how `status` does in the table.

File: web/gds-user-ui/src/modules/dashboard/member/components/MemberNetworkSelect.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (daniellemaxwell) [reply web/gds-user-ui/src/modules/dashboard/member/components/MemberNetworkSelect.tsx:3]: Thanks!

**Pull request merged**